docs/content/docs/build/packages.md

```diff
@@ -1,14 +1,72 @@
+public API for the Fluid Framework, and a service-specific client package, such as the
+`@fluidframework/tinylicious-client` package.
 
-Experimental scope and packages.
+## Primary API: fluid-framework
 
-List other scopes and their meaning (or link to the wiki).
+The `fluid-framework` package bundles a collection of Fluid Framework client packages for easy use when paired with a
+corresponding service client package, such as the `@fluidframework/azure-client` package. The `fluid-framework` package
+consists primarily of two portions: the `IFluidContainer` interface and a selection of distributed data structures
+(DDSes).
+
+### IFluidContainer
+
+The [IFluidContainer][] interface is the one of
+the types returned by calls to `createContainer()` and `getContainer()` on the service clients such as `AzureClient`. It
+includes functionality to retrieve the Fluid data contained within a container, as well as to inspect the state of the
+collaboration session connection.
+
+### DDS packages
+
+You'll use one or more DDSes in your container to model your collaborative data.  The `fluid-framework` package includes
+three data structures that cover a broad range of scenarios:
+
+1. [SharedMap][], a map-like data structure for storing key/value pair data.
+2. [SharedDirectory][], a map-like data structure with ability to organize keys into subdirectories.
+3. [SharedString][], a data structure for string data.
+
+## Package scopes
+
+Fluid Framework packages are published under one of the following npm scopes:
+
+- @fluidframework
+- @fluid-experimental
+- @fluid-internal
+- @fluid-tools
+
+Unless you are contributing to the Fluid Framework, you should only need packages from the **@fluidframework** scope.
+You can [read more about the scopes and their intent][scopes] in the Fluid Framework wiki.
+
+[scopes]: https://github.com/microsoft/FluidFramework/wiki/npm-package-scopes
+
+<!-- AUTO-GENERATED-CONTENT:START (INCLUDE:path=docs/_includes/links.md) -->
+<!-- Links -->
+
+<!-- Concepts -->
+
+[Fluid container]: {{< relref "containers.md" >}}
+
+<!-- Classes and interfaces -->
+
+[ContainerRuntimeFactoryWithDefaultDataStore]: {{< relref "containerruntimefactorywithdefaultdatastore.md" >}}
+[DataObject]: {{< relref "dataobject.md" >}}
+[DataObjectFactory]: {{< relref "dataobjectfactory.md" >}}
+[IFluidContainer]: {{< relref "ifluidcontainer.md" >}}
+[PureDataObject]: {{< relref "puredataobject.md" >}}
+[PureDataObjectFactory]: {{< relref "puredataobjectfactory.md" >}}
+[SharedCounter]: {{< relref "/docs/data-structures/counter.md" >}}
+[SharedMap]: {{< relref "/docs/data-structures/map.md" >}}
+[SharedNumberSequence]: {{< relref "sequences.md#sharedobjectsequence-and-sharednumbersequence" >}}
+[SharedObjectSequence]: {{< relref "sequences.md#sharedobjectsequence-and-sharednumbersequence" >}}
+[SharedSequence]: {{< relref "sequences.md" >}}
+[SharedString]: {{< relref "string.md" >}}
+[TaskManager]: {{< relref "/docs/data-structures/task-manager.md" >}}
```

Comment:
What's this auto-generated from? Most of these things are not included in the fluid-framework package

Reply:
See the reference at the top of the links section. These are "global links."